src/components/CmdKMenu.tsx

```diff
@@ -767,9 +794,35 @@ const CmdKMenu = () => {
         </DialogContent>
       </Dialog>
 
-      <JLCPCBImportDialog
+      <ImportComponentDialog
         open={isJLCPCBDialogOpen}
         onOpenChange={setIsJLCPCBDialogOpen}
+        onComponentSelected={async (component: any) => {
+          try {
+            if (component?.source === "jlcpcb") {
+              const partNumber = component.partNumber || component.name
+              const response = await axios.post("/packages/generate_from_jlcpcb", {
```

Comment:
this api is deprecated, not sure what this is doing here, the whole point was to remove this